src/main/resources/db/migration/V2__text_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE text (
+    uuid UUID PRIMARY KEY,
+    text VARCHAR NOT NULL,
+    coop VARCHAR NOT NULL,
+    key VARCHAR NOT NULL,
+    lang VARCHAR NOT NULL
+);
```

Comment:
```suggestion
);

CREATE INDEX text_coop_idx ON text(coop);
CREATE INDEX text_key_idx ON text(key);
CREATE INDEX text_lang_idx ON text(lang);

CREATE UNIQUE INDEX text_search_idx ON text(coop, key, lang);
```

Since we are searching by `coop`, `key` and `lang`, it would be good to create column indices for those three separately and one for all three of them. That way fetch from database will be more efficient.

I suppose that (`coop`, `key`, `lang`) triple should be unique for each row in the table, so having unique index for that would add the constraint we need.